src/mbi/callbacks.py

```diff
@@ -56,16 +56,23 @@ def default(
     frequency: int = 50,
 ) -> Callback:
     """Creates a default Callback with standard loss functions (L1/L2 Loss/Error, Primal Feas)."""
-    loss_fns = {}
     # Measures distance between input marginals and noisy marginals.
-    loss_fns["L2 Loss"] = marginal_loss.from_linear_measurements(
-        measurements, norm="l2", normalize=True
-    )
-    loss_fns["L1 Loss"] = marginal_loss.from_linear_measurements(
-        measurements,
-        norm="l1",
-        normalize=True,
-    )
+    loss_fns_items = [
```

Comment:
This is less readable, please revert it to the original implementation.  Same for here and below